### ISSUE TITLE
Adding `BVLinearGradient.xcodeproj` to Xcode project crashes Xcode

### ISSUE THREAD

**Issue by wardoost** [react-native-community/react-native-linear-gradient#209]
Installing with `npm install react-native-linear-gradient --save` and `react-native link` works perfectly when starting up with my project `react-native run-ios` but I can't open my Xcode project anymore after that. I'm on RR 0.46.3 and use Xcode 8.3.3.

I also tried to [manually link](https://github.com/react-native-community/react-native-linear-gradient#manually) this module but Xcode crashes when adding `BVLinearGradient.xcodeproj` to `Libraries`. I tried this with versions 2.0.0, 2.0.1 and 2.2.0. Also tried this with Xcode 9.0 beta 5 so it's probably not Xcode 😄 

Anybody else had this before? Could it be a setting in my current project?

**Comment by varunrayen** [comment]
Yes, `react-native link` actually break the Xcode project file. Follow the manual steps and restart the packager and run the app. It should work!

**Comment by SirVeryBritish** [comment]
I also got issues. First I tried with` npm install react-native-linear-gradient --save` and `react-native link`, no success. So I checked if everything was correctly configured like it should be in the manual steps and to my surprise  it seems that 'react-native link' actually configured everything alright. The error, which I got was:
<img width="1661" alt="screen shot 2017-08-19 at 15 23 37" src="https://user-images.githubusercontent.com/10279069/29486988-94ca1a4e-84f2-11e7-8d36-8be7137ce140.png">
Okay 'react-native link' is anyway not recommended by a lot of users. So I try it manually. Step by step, clear, rebuild aaaand the same issue. Just in case I'm including my header search paths for BVLinearGradient.
<img width="849" alt="screen shot 2017-08-19 at 15 30 24" src="https://user-images.githubusercontent.com/10279069/29487046-6c0f1b76-84f3-11e7-8fcc-a3644ee768c4.png">
